src/api/status/src/server.js

```diff
@@ -48,6 +49,17 @@ const satelliteOptions = {
       },
     },
   },
+  healthCheck: async () => {
+    const sha = process.env.GIT_COMMIT || 'master';
+    const gitHubUrl = `https://github.com/Seneca-CDOT/telescope/commit/${sha}`;
+    const { version } = await getPackage('https://github.com/Seneca-CDOT/telescope/blob/master/');
```

Comment:
Can we cache this result?  Will it call it every time we hit the health check?  Seems like we should do it once only.